zubhub_frontend/zubhub/src/views/PageWrapper.jsx

```diff
@@ -118,6 +118,23 @@ function PageWrapper(props) {
     };
   }, [handleScroll]);
 
+  const unprotectedRoutes = [
```

Comment:
created an array for the unprotected routes

Reply:
what is this for?

Reply:
Ahh! I see. Thanks